### PR TITLE
DOCS: Update Android docs to match new touch controls settings

### PR DESCRIPTION
**Pull request by lephilousophe**

### CODE REVIEW

File: doc/docportal/other_platforms/android.rst [hunk 1]
```diff
@@ -86,20 +86,25 @@ Controls can also be manually configured in the :doc:`Keymaps tab <../settings/k
...
+The touch control scheme can be configured in the global settings. From the Launcher, go to **Options > Backend > Choose the preferred touch mode**.
+It's possible to configure the touch mode for three situations (ScummVM menus, 2D games and 3D games) and choose one of the three possible modes:
 
-    - When touchpad mouse mode is off (default), the touch controls are direct. The pointer jumps to where the finger touches the screen.
-    - When touchpad mouse mode is on, the touch controls are indirect. The finger can be far away from the pointer and still move it, like on a laptop touchpad.
+    - Direct mouse, the touch controls are direct. The pointer jumps to where the finger touches the screen (default for menus).
+    - Touchpad emulation, the touch controls are indirect. The finger can be far away from the pointer and still move it, like on a laptop touchpad.
+    - Gamepad emulation; the touch control don't move any mouse. The fingers must be placed on lower left and right of the screen and respectively emulate a directional pad and action buttons.
```
Comment (antoniou79) [inline doc/docportal/other_platforms/android.rst:101]: "Gamepad emulation; the touch control don't move any mouse" -> "Gamepad emulation, the touch controls don't move any mouse"
ie. I suggest to change the ";" to "," to be similar with the previous bullet points, and change "control" to "controls".

File: doc/docportal/other_platforms/android.rst [hunk 2]
```diff
@@ -86,20 +86,25 @@ Controls can also be manually configured in the :doc:`Keymaps tab <../settings/k
...
     - The pointer speed setting in the :doc:`Controls tab <../settings/control>` affects how far the pointer moves in response to a finger movement.
 
+The touch mode can be switched at anytime by tapping on the controller icon, next to the menu icon at the top right of the screen.
+
+To display or hide the small controller icon, from the Launcher select **Options** and then the **Backend** tab. Tick the **Show on-screen control** box to enable the keyboard icon.
```
Comment (antoniou79) [inline doc/docportal/other_platforms/android.rst:106]: The end of this paragraph seems outdated. If the "Show on-screen control" box is ticked then we enabled to small controller icon overlay. If unticked, it is hidden. The keyboard icon is no longer displayed (unless the checkbox is ticked and we are at a text field or explicitly called the popup keyboard with the long press)

**Review by antoniou79 — COMMENTED** [review]
Looks good. I've added a couple of comments for consideration.

File: doc/docportal/other_platforms/android.rst (hunk 1 above)
Comment (lephilousophe) [reply doc/docportal/other_platforms/android.rst]: Indeed it was typos.

File: doc/docportal/other_platforms/android.rst (hunk 2 above)
Comment (lephilousophe) [reply doc/docportal/other_platforms/android.rst]: Yes, I saw this but forgot to git add before committing...
Thanks!

**Comment by lephilousophe** [comment]
Merging. Thanks.

**Pull request merged**